kv/mock_test.go

```diff
@@ -56,7 +56,8 @@ func (s testMockSuite) TestInterface(c *C) {
 		_, err = transaction.IterReverse(Key("lock"))
 		c.Check(err, IsNil)
 	}
-	transaction.Commit(context.Background())
+	// to fix: cannot assert err, e.g. KV error safe to retry %s
+	_ = transaction.Commit(context.Background())
```

Comment:
PTAL @tiancaiamao

Reply:
This code just check the API interface, even if here returns an error, it doesn't matter.